# ISOTOPE
ISOform-guided prediction of epiTOPEs in cancer

The following pipeline have been developed for the identification of cancer-specific splicing-derived epitopes from RNA-seq. The first input of this method are the read counts mapped to all posible junctions in the genome. This file is created through Junckey (https://github.com/comprna/Junckey#1-format-star-output).

The pipeline is divided in 4 parts, depending the event type the user wants to obtain:

   * Pseudoexons (Exonizations)
   * New exons skipping events (Neoskipping)
   * Alternative splice site (A5_A3)
   * Intron retention (IR)
   
The user must run each of the 3 parts sequentially. The scripts are ready to be run in a slurm cluster. Until all jobs generated by a part are finished do not run the following part. For the moment, the user needs to manually change in the code the input files. In the near future we will addecuate the pipeline for entering the files by command.

For any query related with the tool, please create an issue in the repository. 
